### PR TITLE
Upgrade to fx69

### PR DESCRIPTION
**Pull request by motin**
Fixes https://github.com/mozilla/OpenWPM/issues/454

Should probably not be merged before Firefox 69 is officially released.

### CODE REVIEW

**Comment by englehardt** [comment]
@motin is there a reason to keep the old build links around but commented out? I prefer to remove them to keep the files clean, but I want to make sure you don't have a reason for keeping them around.

**Comment by englehardt** [comment]
@nhn11 mind to do a quick review, since this PR is a combined effort from me and @motin :)

**Comment by englehardt** [comment]
These test failures should be fixed by #490, so it's safe to ignore them.

File: automation/Extension/firefox/src/privileged/sockets/api.js [hunk 1]
```diff
@@ -112,7 +112,7 @@ this.sockets = class extends ExtensionAPI {
 
           try {
             let socket = gManager.sendingSocketMap.get(id);
-            var transport = socketService.createTransport(null, 0, host, port, null);
+            var transport = socketService.createTransport([], host, port, null);
```
Comment (nhnt11) [inline automation/Extension/firefox/src/privileged/sockets/api.js]: Nice, thanks for figuring this out.

**Pull request merged**